CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Added
+
+- Add network timeout setting - [#324](https://github.com/owncloud/user_ldap/issues/324)
+
+### Changed
+
+- Allow avatars to be changed by users if not provided by LDAP - [#188](https://github.com/owncloud/user_ldap/issues/188)
+- Remove PHP 5.6 support - [#388](https://github.com/owncloud/user_ldap/issues/388)
+- LDAP configuration is now stored JSON encoded - [#393](https://github.com/owncloud/user_ldap/issues/393)
+- Clean up Application initialization code - [#396](https://github.com/owncloud/user_ldap/issues/396)
+- Remove unused use statements - [#399](https://github.com/owncloud/user_ldap/issues/399) [#400](https://github.com/owncloud/user_ldap/issues/400)
+
+### Fixed
+
+- Do not return unavailable users - [#12](https://github.com/owncloud/user_ldap/issues/12)
```

Comment:
Maybe "only return users valid for ownCloud when getting LDAP group members"
The issue is that there can be a filter in ownCloud that reduces the set of users from the LDAP server that are allowed to use ownCloud. But their might be an LDAP group that has a mix of users that are and are not in the filter for ownCloud. So when the user_ldap app asks for a list of the members in the group, we just want to find the ones that are valid for ownCloud.

Reply:
adjusted. thanks.

I didn't have enough insight about what this one was about